### PR TITLE
Update docker image

### PR DESCRIPTION
**Pull request by DeployInBinary**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ LABEL Description="Tyk Dashboard docker image" Vendor="Tyk" Version=$TYKVERSION
 RUN apt-get update \
  && apt-get upgrade -y \
  && apt-get install -y --no-install-recommends \
-            curl ca-certificates apt-transport-https \
+            curl ca-certificates apt-transport-https gnupg npm \
```
Comment (excieve) [inline Dockerfile]: Sorry, why is this needed? Also `npm` should probably be installed further down after adding the custom nodejs repo. In addition to this worth uninstalling `npm` in the end as it's not really needed for the container to run I think.

File: Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ LABEL Description="Tyk Dashboard docker image" Vendor="Tyk" Version=$TYKVERSION
...
             build-essential \
  && curl -L https://packagecloud.io/tyk/tyk-dashboard/gpgkey | apt-key add - \
  && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
```
Comment (excieve) [inline Dockerfile:14]: I suggest updating this too, nodejs 8.x is way too old. Let's go with 10.x instead if it doesn't error out when installing aglio.

**Review by excieve — CHANGES_REQUESTED** [review]
Some minor comments.

**Review by excieve — CHANGES_REQUESTED** [review]
As discussed previously, please remove nodejs if aglio is decided to be deprecated from the official image (after you get confirmation). It's not needed for anything else so would conserve quite a bit of space.

**Comment by excieve** [comment]
@vesko-tyk We might want to split this PR in two.

It appears we're going to deprecate API Blueprint rendering support out of the box in this image. This means the following:

  1. In the next dashboard release the release notes must mention it being deprecated in some future release (e.g. 1.10)
  2. Prepare a PR to do this (removing aglio and nodejs from the image)
  3. Merge it for the release when deprecation is to be enforced (e.g. 1.10)

FYI @buger 

So the base image update PR would have to be separate, not touching aglio or nodejs for now.

**Comment by DeployInBinary** [comment]
@excieve Ideally I would like to leave this PR like that and update the docker image together with deprecating the API Blueprint feature. If I separate the PR and do the docker image upgrade then there are a lot of issues and incompatibilities between the newer version of nodejs (which we should use, if we update the base image to debian:buster) and aglio, aglio is not maintained for more than 2 years now and works only with nodejs v8.x

**Comment by excieve** [comment]
@vesko-tyk We can keep old nodejs for the time being. Deprecation isn't going to happen until at least 2.10 or later since API Blueprint might be used in the wild and appropriate notice must be given. Base image update shouldn't depend on this or it will be delayed until much later next year.

**Comment by DeployInBinary** [comment]
@excieve Not a problem at all. I have now updated this PR to include only the upgrade of the base docker image we use and to keep older version of nodejs, so aglio can work. I will create new PR as soon as this one is merged to reflect the deprecation of API Blueprint and removal of old nodejs version and aglio.

**Review by excieve — APPROVED** [review]
LGTM, thanks @vesko-tyk!

**Pull request merged**